.github/workflows/gradle-build-test.yaml

```diff
@@ -0,0 +1,44 @@
+name: Build and Test Github-Metrics
+
+on:
+    push:
+        branches: [ "main" ]
+    pull_request:
+        branches: [ "main" ]
+
+jobs:
+    build:
+
+        runs-on: ubuntu-latest
+        permissions:
+            contents: read
+
+        steps:
+            -   uses: actions/checkout@v4
+            -   name: Set up JDK 21
+                uses: actions/setup-java@v4
+                with:
+                    java-version: '21'
+                    distribution: 'corretto'
+            -   name: Setup Gradle
```

Comment:
i think you dont need this step as you are using the wrapper, I could be wrong though

Reply:
this is now fixed